db/migrate/20221219051529_fix_feature_flags.rb

```diff
@@ -0,0 +1,46 @@
+class FixFeatureFlags < ActiveRecord::Migration[6.1]
+  # rubocop:disable Metrics/MethodLength
+  # rubocop:disable Metrics/BlockLength
+  def change
+    installation_config = InstallationConfig.find_by(name: 'DEPLOYMENT_ENV')
+    Account.find_in_batches do |account_batch|
+      account_batch.each do |account|
+        if installation_config == 'cloud'
+          account.enable_features(
+            'reports',
+            'crm',
+            'auto_resolve_conversations'
+          )
+        else
+          account.enable_features(
```

Comment:
Wouldn't this rewrite the configuration in self-hosted accounts where a user might have already saved some configurations to existing accounts?